packages/calcite-components/src/components/slider/slider.stories.ts

```diff
@@ -58,412 +61,609 @@ export default {
   },
 };
 
-export const simple = (args: SliderStoryArgs): string => html`
-  <calcite-slider
-    min="${args.min}"
-    max="${args.max}"
-    value="${args.value}"
-    step="${args.step}"
-    min-label="${args.minLabel}"
-    ${boolean("disabled", args.disabled)}
-    ${boolean("label-handles", args.labelHandles)}
-    ${boolean("label-ticks", args.labelTicks)}
-    ticks="${args.ticks}"
-    page-step="${args.pageStep}"
-    ${boolean("precise", args.precise)}
-    ${boolean("mirrored", args.mirrored)}
-    ${boolean("snap", args.snap)}
-    scale="${args.scale}"
-  ></calcite-slider>
-`;
-
-export const range = (): string => html`
-  <calcite-slider
-    min="0"
-    min-label="Temperature, lower bound"
-    min-value="25"
-    max="100"
-    max-label="Temperature, upper bound"
-    max-value="75"
-    step="1"
-    ticks="20"
-    snap
-    scale="m"
-  ></calcite-slider>
-`;
-
-export const darkModeMirroredRange_TestOnly = (): string => html`
-  <calcite-slider
-    class="calcite-mode-dark"
-    mirrored
-    min="0"
-    min-label="Temperature, lower bound"
-    min-value="25"
-    max="100"
-    max-label="Temperature, upper bound"
-    max-value="75"
-    step="1"
-    label-handles
-    label-ticks
-    ticks="20"
-    precise
-    snap
-    scale="m"
-  ></calcite-slider>
-`;
+const simple = {};
+for (const l of ["horizontal", "vertical"]) {
+  simple[l] = (args: SliderStoryArgs): string => html`
+    <calcite-slider
+      min="${args.min}"
+      max="${args.max}"
+      value="${args.value}"
+      step="${args.step}"
+      min-label="${args.minLabel}"
+      ${boolean("disabled", args.disabled)}
+      ${boolean("label-handles", args.labelHandles)}
+      ${boolean("label-ticks", args.labelTicks)}
+      ticks="${args.ticks}"
+      page-step="${args.pageStep}"
+      ${boolean("precise", args.precise)}
+      ${boolean("mirrored", args.mirrored)}
+      ${boolean("snap", args.snap)}
+      scale="${args.scale}"
+      layout=${l}
+      style="${args.style}"
+    ></calcite-slider>
+  `;
+}
+export const simpleH = simple["horizontal"];
```

Comment:
what if we make it so each story has two sliders - one horizontal and one vertical? That way we don't create double slider's chromatic snapshot count. Specifically for `simple` we only need one slider because it can be changed via knobs like I mentioned above.

Reply:
I dig this. Besides reducing snapshots, it also makes it easier to add coverage for both compared to handling them separately.